new-website/deepchem/pages/layers.js

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useState } from "react";
+
+import Image from "next/image";
+
+import LayerCard from "/components/Layers/LayerCard";
+import FilterButton from "/components/Layers/FilterButton";
+
+import layers from "/data/layers/layers.json";
+import modelList from "/data/layers/models.json";
+
+import deepchemPyTorch from "/public/icons/deepchem-pytorch.png";
+import deepchemKeras from "/public/icons/deepchem-keras.png";
+import deepchemFilter from "/public/icons/deepchem-filter.png";
+
+
+/**
+ * Models component that displays the models page of the application
+ * @component
+ * @return {JSX.Element} The JSX element to render the Model component
+ */
+const Layers = () => { 
+
+  const [filteredLayers, setFilteredLayers] = useState(layers);
+  const [models, setModels] = useState([]);
```

Comment:
I am not sure what models exactly are in the layers page, is the naming intended?

Reply:
Models that use the layers so layers can be filtered with their respective models.